dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -480,6 +480,10 @@ boolean catchesLinkageError(String sourceClassName) {
       ClassGen classGen = new ClassGen(sourceJavaClass);
 
       for (Method method : sourceJavaClass.getMethods()) {
+        if (method.getCode() == null) {
+          // No need to check the presence of try-catch clause for methods without code
+          continue;
```

Comment:
would it be cleaner to simply wrap the loop body in ` if (method.getCode() != null) `?

Reply:
If you ask me, I prefer fewer nested code  (less block indentation).